rpc/wrtc_signaling_answerer.go

```diff
@@ -210,8 +227,10 @@ func (ans *webrtcSignalingAnswerer) Stop() {
 	ans.startStopMu.Lock()
 	defer ans.startStopMu.Unlock()
 
-	ans.cancelBackgroundWorkers()
-	ans.activeBackgroundWorkers.Wait()
+	ans.cancelAnswerWorkers()
+	ans.answerMu.Lock()
+	ans.answerWorkers.Wait()
+	ans.answerMu.Unlock()
```

Comment:
super minor - noticed this flow is ever-so-slightly different in the corresponding [caller PR](https://github.com/viamrobotics/goutils/pull/265/files#r1526670921). is that intentional? should we standardize?

Reply:
Not intentional! I'll standardize to the `defer`.